framework/lib/components/createable-select-dropdown/createable-select-dropdown.tsx

```diff
@@ -105,6 +144,12 @@ export const CreatableSelectDropdown = ({
     }
   }
 
+  useEffect(() => {
+    if (initialValue) {
+      setSelectedOption(standardOptions.find((option) => option.value === initialValue) || null)
```

Comment:
Don't use a useEffect for this. Set this as the initial state
You can use `useMemo` if you dont want it to search on every render